portfolio/src/main/webapp/index.html

```diff
@@ -51,7 +47,96 @@ <h1>GENERAL</h1>
+                            <p> My favourite photo of lake Ontario taken near my home </p>
+                        </div>
+                    </div>
+                    <div class="gridItem" style="background-image: url('images/gallery/trouble-1.jpg')">
+                        <div class="caption">
+                            <p> This is one of my cats, her name is Trouble </p>
+                        </div>
+                    </div>
+                    <div class="gridItem" style="background-image: url('images/gallery/edi-11.jpg')">
+                        <div class="caption">
+                            <p> The view of Edinburgh from the Salisbury Crags </p>
+                        </div>
+                    </div>
+                    <div class="gridItem" style="background-image: url('images/gallery/swiss-2.JPG')">
+                        <div class="caption">
+                            <p> A birds-eye view of Wengen from Mänlichen, Switzerland </p>
+                        </div>
+                    </div>
+                    <div class="gridItem" style="background-image: url('images/gallery/edi-3.jpg')">
+                        <div class="caption">
+                            <p> Edinburgh from near the top of Arthur's seat </p>
+                        </div>
+                    </div>
+                    <div class="gridItem" style="background-image: url('images/gallery/swiss-1.JPG')">
+                        <div class="caption">
+                            <p> Near Lauterbrunnen, Switzerland </p>
+                        </div>
+                    </div>
+                    <div class="gridItem" style="background-image: url('images/gallery/edi-4.jpg')">
+                        <div class="caption">
+                            <p> Climate Strike in the Meadows, Edinburgh </p>
+                        </div>
+                    </div>
+                    <div class="gridItem" style="background-image: url('images/gallery/shadow-1.jpg')">
+                        <div class="caption">
+                            <p> This is my other cat named Shadow </p>
+                        </div>
+                    </div>
+                    <div class="gridItem" style="background-image: url('images/gallery/edi-5.jpg')">
+                        <div class="caption">
+                            <p> Princes street at night in Edinburgh </p>
+                        </div>
+                    </div>
+                    <div class="gridItem" style="background-image: url('images/gallery/Nice-1.JPG')">
+                        <div class="caption">
+                            <p> Beautiful view of Nice from Parc du Mont Boron </p>
+                        </div>
+                    </div>
+                    <div class="gridItem" style="background-image: url('images/gallery/edi-6.jpg')">
+                        <div class="caption">
+                            <p> Edinburgh or Hogsmeade? </p>
```

Comment:
That's exactly what I thought when I went on a trip to Ediburgh! Fun fact is that I had no idea that JK Rowling wrote Harry Potter in Edinburgh. I was randomly walking when I ran into the elephant house and learned about it.

Reply:
Me too! It's one of the many reasons I love the city. The fact that half the cafes have "The birthplace of Harry Potter" written on their windows gave it away😆